projects/saturn/src/main/java/io/fairspace/saturn/webdav/DirectoryResource.java

```diff
@@ -208,6 +208,8 @@ private void uploadFile(String path, FileItem file) throws NotAuthorizedExceptio
     }
 
     private void uploadMetadata(FileItem file) throws BadRequestException {
+        var entityColumn = "Name";
```

Comment:
We have to rethink this, because in case of sample entity there are two "name" columns now -one added by the template generator and one from the model (sample has a "name" property defined in the model).